mkdocs.yml

```diff
@@ -188,7 +188,7 @@ nav:
           - Mcrouter: sandbox/apps/mcrouter.md
           - OliveTin: sandbox/apps/olivetin.md
           - Reposilite: sandbox/apps/reposilite.md
-          - SepahoreUI: sandbox/apps/semasoreui.md
+          - SepahoreUI: sandbox/apps/semaphoreui.md
```

Comment:
Title is wrong.

Reply:
Are you referring to the SemaphoereUI or the location after it?